packages/custom_lint/test/cli_test.dart

```diff
@@ -106,12 +107,66 @@ lib/custom_lint_client.dart:14:29: Error: Undefined name 'createPlugin'.
         expect(err, emitsDone);
         expect(
           out.join(),
-          completion('''
-  lib/another.dart:1:6 • Hello world • hello_world
-  lib/another.dart:1:6 • Oy • oy
-  lib/main.dart:1:6 • Hello world • hello_world
-  lib/main.dart:1:6 • Oy • oy
+          completion(
+            allOf(
+              matchIgnoringAnsi(contains, '''
+   info ${ansi.bullet} lib/another.dart:1:6 ${ansi.bullet} Hello world ${ansi.bullet} hello_world
```

Comment:
Why use ${ansi.bullet}?

Reply:
It defaults to `-` on non-ansi terminals. This usage comes straight from the Dart analyzer.